lib/commands/viewport.js

```diff
@@ -0,0 +1,33 @@
+
+import { ImageHelpers } from 'appium-base-driver';
+
+let extensions = {}, commands = {};
+
+commands.getStatusBarHeight = async function () {
+  return await this.uiautomator2.jwproxy.command(`/appium/device/stat_bar_height`, 'GET', {});
```

Comment:
[/wd/hub/session/:sessionId/appium/device/system_bars](https://github.com/appium/appium-base-driver/blob/8481b9ac7cfa89563e86bc9f53e1b1d0127f6d6b/lib/mjsonwp/routes.js#L418)

Reply:
I wasn't aware of this link. Notice however, that this route is not implemented for appium-ios-driver, while ours is implemented for both Android and iOS. Should we still use this route?

Reply:
@vmaxim @imurchie  ?

Reply:
somehow I wasn't aware of this route either. But given that it exists we should probably use it instead of creating another route. @danielputerman it looks possible to simply update the return value to conform to the shape of `system_bars`, right?